announcements/_posts/2020-09-18-w05-latex.md

```diff
@@ -10,7 +10,7 @@ location: "Discord"
 Hey RITlug!
 
 
-In preparation for the upcoming virtual career fair, this week we're covering LaTeX! LaTeX is a great tool for building tons of professional documents, including resumes. Come on by to our discord where Jeffrery Russel will be giving a talk on the basics of LaTeX and how you can use it to supercharge your resume.
+In preparation for the upcoming virtual career fair, this week we're covering LaTeX! LaTeX is a great tool for building tons of professional documents, including resumes. Come on by to our discord where Jeffrery Russell will be giving a talk on the basics of LaTeX and how you can use it to supercharge your resume.
```

Comment:
*Jeffery * 😃

Reply:
I'm beginning to wonder why I accepted a position as secretary when I'm actually illiterate

Reply:
Lol, you are doing great! RITlug eboard has a rich legacy of fixing englishfd in PRs